app/src/main/java/co/co/nimblehq/showcases/poly/place/SliceProvider.kt

```diff
@@ -0,0 +1,101 @@
+package co.co.nimblehq.showcases.poly.place
+
+import android.content.ContentResolver
+import android.content.Intent
+import android.net.Uri
+import androidx.slice.Slice
+import androidx.slice.SliceProvider
+import androidx.slice.builders.ListBuilder
+import androidx.slice.builders.SliceAction
+
+class SliceProvider : SliceProvider() {
+    /**
+     * Instantiate any required objects. Return true if the provider was successfully created,
+     * false otherwise.
+     */
+    override fun onCreateSliceProvider(): Boolean {
+        return true
+    }
```

Comment:
I think we can shorten it to be like this for shorter code 😄 
```suggestion
    override fun onCreateSliceProvider() = true
```